main.go

```diff
@@ -90,12 +90,19 @@ func main() {
 		setupLog.Error(err, "unable to get config")
 		os.Exit(1)
 	}
-
+	log, err := logr.NewLoggerWithOptions(
+		logr.WithOptionsFromEnv(),
+		logr.WithComponentName("vcluster-controller"),
+	)
+	if err != nil {
+		setupLog.Error(err, "unable to setup logger")
+		os.Exit(1)
+	}
```

Comment:
Some spacing around the top and bottom
```suggestion

	log, err := logr.NewLoggerWithOptions(
		logr.WithOptionsFromEnv(),
		logr.WithComponentName("vcluster-controller"),
	)
	if err != nil {
		setupLog.Error(err, "unable to setup logger")
		os.Exit(1)
	}

```